Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGalleryCell.swift

```diff
@@ -67,26 +60,34 @@ struct BasemapGalleryCell: View {
         }).disabled(item.isBasemapLoading)
     }
     
-    /// Creates a red exclamation mark, used to denote a basemap with an error.
-    /// - Returns: A new transparent rectagle view.
-    private func makeErrorOverlay() -> some View {
-        HStack {
-            Spacer()
-            VStack{
-                ZStack {
-                    // For a white background behind the exclamation mark.
-                    Circle()
-                        .foregroundColor(.white)
-                        .frame(width: 16, height: 16)
-                    Image(systemName: "exclamationmark.circle.fill")
-                        .foregroundColor(.red)
-                        .background(Color.clear)
-                        .frame(width: 32, height: 32)
+    /// Creates an overlay which is either a selection outline or an error icon.
+    /// - Returns: A thumbnail overlay view.
+    private func makeOverlay() -> some View {
+        Group {
+            if item.hasError {
+                HStack {
+                    Spacer()
+                    VStack{
+                        ZStack {
+                            // For a white background behind the exclamation mark.
+                            Circle()
+                                .foregroundColor(.white)
+                                .frame(width: 16, height: 16)
+                            Image(systemName: "exclamationmark.circle.fill")
+                                .foregroundColor(.red)
+                                .background(Color.clear)
```

Comment:
I diffed with/without this line, not sure that I see any difference. May I be missing something?

Reply:
I removed it; the default is probably `.clear`.